app/ui.js

```diff
@@ -1043,6 +1043,8 @@ const UI = {
         UI.rfb.addEventListener("clipboard", UI.clipboardReceive);
         UI.rfb.addEventListener("bell", UI.bell);
         UI.rfb.addEventListener("desktopname", UI.updateDesktopName);
+        UI.rfb.addEventListener("rsakeyget", UI.getRSAKey);
+        UI.rfb.addEventListener("rsakeyset", UI.setRSAKey);
```

Comment:
These are used for authentication, so I think these could be handled by the credentials mechanism we already have.